s64da_benchmark_toolkit/streams.py

```diff
@@ -82,6 +83,8 @@ def apply_sql_modifications(sql, modifications):
         return sql
 
     def run(self):
+        if not is_netdata_set_and_runnning(self.config):
+            return
```

Comment:
Is this intentional?

Reply:
Yes. The thinking behind it was that if you're running the toolkit and expecting to also get netdata data collected, you'll want to know if this is not going to happen ASAP and not only at the end of the run.
However, this was assuming that you explicitly asking the tool to collect netdata data. If netdata data is collected by default, then this is not required here.